tests/components/enphase_envoy/test_select.py

```diff
@@ -71,106 +68,95 @@ async def test_no_select(
 @pytest.mark.parametrize(
     ("mock_envoy"), ["envoy_metered_batt_relay"], indirect=["mock_envoy"]
 )
+@pytest.mark.parametrize(
+    ("relay", "target"),
```

Comment:
Since the individual values of `target` are used within the test, it would be better to separate them into 3 descriptive parameters instead of passing a tuple (which is incorrectly typed as `str` in the test function signature) below.